app/experimenter/nimbus-ui/src/components/ConfidenceInterval/index.tsx

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* istanbul ignore file */
+// TODO: remove this ^
+
+import React from "react";
+
+import { SIGNIFICANCE } from "../../lib/visualization/constants";
+
+const renderBounds = (lower: number, upper: number, significance: string) => {
+  const padding = <div className="col p-0" />;
+  const numbers = (
+    <>
+      <div
+        className={`${significance}-significance col text-left p-0 h6 font-weight-normal`}
+      >
+        {lower}%
+      </div>
+      <div
+        className={`${significance}-significance col text-right p-0 h6 font-weight-normal`}
+      >
+        {upper}%
+      </div>
+    </>
+  );
+
+  switch (significance) {
+    case SIGNIFICANCE.NEGATIVE:
+      return (
+        <>
+          {numbers}
+          {padding}
+          {padding}
+        </>
+      );
+    case SIGNIFICANCE.POSITIVE:
+      return (
+        <>
+          {padding}
+          {padding}
+          {numbers}
+        </>
+      );
+    case SIGNIFICANCE.NEUTRAL:
+      return (
+        <>
+          {padding}
+          {numbers}
+          {padding}
+        </>
+      );
+  }
+};
```

Comment:
It seems to me that these could just be strings and not JSX?

Reply:
`numbers` is a JSX element and isn't a string.

Reply:
nvvvvm, `numbers` is JSX, my bad

Reply:
![4mvtwy](https://user-images.githubusercontent.com/6392049/99559624-feb26400-2992-11eb-82dc-e2f60ff9cba9.jpg)